packages/smooth_app/lib/database/dao_secured_string.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:hive_flutter/hive_flutter.dart';
+import 'package:smooth_app/database/abstract_dao.dart';
+import 'package:smooth_app/database/local_database.dart';
+
+enum SecuredValues {
+  USER_ID,
+  PASSWORD,
+}
```

Comment:
Probably useless here: the dao works as a map, and we're not supposed to tell it in advance all the possible keys.